app/components/Speaker/index.scss

```diff
@@ -0,0 +1,106 @@
+.Speaker {
+  width: 100%;
+  margin: 25px 0px 30px 0px;
+
+  @media (min-width: 770px) {
+    width: 48%;
+  }
+
+  @media (min-width: 1000px) { // eyeballed
+    width: 30%;
+  }
+
+  &__name {
+    margin: 0px;
+    font-size: 18px;
+    font-weight: normal;
+    &--bold {
+      font-weight: bold;
+    }
+  }
+
+  &__title {
+    margin: 8px 0px;
+    font-size: 20px;
+    font-weight: 600;
+  }
+
+  &__summary {
+    margin: 0px;
+  }
+
+  a {
+    font-weight: bold;
+    text-decoration: underline;
+    color: #4A4A4A;
+  }
+
+  &__photo {
+    width: 120px;
+    height: 120px;
+    margin: 0px;
+    overflow: hidden;
+    border: $solid-border;
+    border-radius: 50%;
+    text-align: center;
+
+    display: -webkit-flex;
```

Comment:
Why do we have these -webkit prefixes everywhere? We should be using autoprefixer